arches_her/media/css/report_templates.css

```diff
@@ -506,14 +506,14 @@ h3.shim:hover {
     padding-bottom: 50px;
 }
 
-.aher-block-attributes h4 {
+.aher-block-attributes h4, .aher-block-attributes span {
     font-size: 15px;
     font-weight: 400;
     margin: 2px 0px;
     display: inline-block;
 }
 
-.aher-block-attributes h6 {
+.aher-block-attributes h6, .aher-block-attributes .h6 {
```

Comment:
It looks unnecessary, unless there is a class called h6

Reply:
Removed, as it does appear to be unnecessary now.